dot/rpc/modules/test_data/test_metadata.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package testdata
+
+import (
+	"archive/zip"
+	"bytes"
+	_ "embed"
+	"io"
+)
+
+//go:embed test_metadata.zip
+var testMetaData []byte
+
+func init() {
+	bytesReader := bytes.NewReader(testMetaData)
+	zipReader, err := zip.NewReader(bytesReader, int64(bytesReader.Len()))
+	if err != nil {
+		panic(err)
+	}
+
+	zipFile := zipReader.File[0]
+
+	file, err := zipFile.Open()
+	if err != nil {
+		panic(err)
+	}
+
+	defer func() {
+		err := file.Close()
+		if err != nil {
+			panic(err)
+		}
+	}()
+
+	b, err := io.ReadAll(file)
+	if err != nil {
+		panic(err)
+	}
+
+	testData = string(b)
+}
+
+var testData string
+
+// GetTestData returns the testData string.
+func GetTestData() string {
```

Comment:
what kind of test data is this? Maybe rename to be more explicit and follow `New....` syntax.

Reply:
I suggested this. We just get the test data from the test_data package. But I'm fine with `New...` as well.
And yes naming the `TestData` to something more specific would be nice.